### PR TITLE
Move to F35

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: manifests/fedora-coreos-base.yaml [hunk 1]
```diff
@@ -185,3 +191,9 @@ packages-aarch64:
 arch-include:
   x86_64: bootupd.yaml
   aarch64: bootupd.yaml
+
+remove-from-packages:
+  # Hopefully short-term hack -- see https://github.com/coreos/fedora-coreos-config/pull/1206#discussion_r705425869.
+  # This keeps the size down and ensures nothing tries to use it, preventing us
+  # from shedding the dep eventually.
+  - [cracklib-dicts, .*]
```
Comment (dustymabe) [inline manifests/fedora-coreos-base.yaml:199]: @jlebon, any followups we need to do here? would like to not carry this forever

Comment (jlebon) [reply manifests/fedora-coreos-base.yaml:199]: The latest on this is https://bugzilla.redhat.com/show_bug.cgi?id=2006063#c4.

**Review by jlebon — APPROVED** [review]
LGTM!

File: manifests/fedora-coreos-base.yaml (hunk 1 above)
Comment (dustymabe) [reply manifests/fedora-coreos-base.yaml:199]: I wonder if that feature needs to be formally requested somewhere (else it will just be forgotten)?

**Comment by dustymabe** [comment]
```
[2021-11-04T22:09:33.068Z] Downgraded:
[2021-11-04T22:09:33.068Z]   kernel 5.14.14-200.fc34 -> 5.14.13-300.fc35
[2021-11-04T22:09:33.068Z]   kernel-core 5.14.14-200.fc34 -> 5.14.13-300.fc35
[2021-11-04T22:09:33.068Z]   kernel-modules 5.14.14-200.fc34 -> 5.14.13-300.fc35
[2021-11-04T22:09:33.068Z] Removed:
[2021-11-04T22:09:33.068Z]   compat-readline5-5.2-39.fc34.x86_64
[2021-11-04T22:09:33.068Z]   iptables-1.8.7-3.fc34.x86_64
[2021-11-04T22:09:33.068Z]   libtextstyle-0.21-4.fc34.x86_64
[2021-11-04T22:09:33.068Z]   libusbx-1.0.24-2.fc34.x86_64
[2021-11-04T22:09:33.068Z]   systemd-rpm-macros-248.9-1.fc34.noarch
[2021-11-04T22:09:33.068Z] Added:
[2021-11-04T22:09:33.068Z]   gdbm-libs-1:1.22-1.fc35.x86_64
[2021-11-04T22:09:33.068Z]   iptables-legacy-1.8.7-13.fc35.x86_64
[2021-11-04T22:09:33.068Z]   iptables-legacy-libs-1.8.7-13.fc35.x86_64
[2021-11-04T22:09:33.068Z]   libusb1-1.0.24-4.fc35.x86_64
[2021-11-04T22:09:33.068Z]   shadow-utils-subid-2:4.9-3.fc35.x86_64
[2021-11-04T22:09:33.068Z]   systemd-resolved-249.4-2.fc35.x86_64
[2021-11-04T22:09:33.068Z]   util-linux-core-2.37.2-1.fc35.x86_64
[2021-11-04T22:09:33.068Z] + grep -q Downgraded tmp/diff.txt
[2021-11-04T22:09:33.068Z] + echo 'Downgrade detected. This is likely unintentional. If not, you may safely ignore this error.'
[2021-11-04T22:09:33.068Z] Downgrade detected. This is likely unintentional. If not, you may safely ignore this error.
```

I'll safely ignore the downgrade. The problem will resolve itself shortly.

**Pull request merged**